### PR TITLE
now using `show` from Compat.jl instead of deprecated writemime

### PR DESCRIPTION
**Pull request by ssfrr**

### CODE REVIEW

**Comment by yuyichao** [comment]
@Keno ping

**Comment by yuyichao** [comment]
@ssfrr Should require Compat 0.8.0

File: src/TexExtensions.jl [hunk 1]
```diff
@@ -104,7 +104,7 @@ else
     grisu_fmt(x, mode, n) = Base.Grisu.grisu(x, mode, n)
 end
 
-function writemime(io::IO, ::T, x::BigFloat)
+function show(io::IO, ::T, x::BigFloat)
```
Comment (yuyichao) [inline src/TexExtensions.jl]: Why is `@compat` not needed here?

Comment (ssfrr) [inline src/TexExtensions.jl]: whoops, should be. fixed.

**Comment by ssfrr** [comment]
Fixed both issues (missing lower-bound on version and missing `@compat` for a `show`), thanks for the review.

**Comment by Keno** [comment]
LGTM

**Pull request merged**